images/ipi-deprovision/Dockerfile

```diff
@@ -3,6 +3,8 @@ LABEL maintainer="hongkliu@redhat.com"
 
 ARG JQ_VERSION=1.6
 
+COPY --from=cli /usr/bin/oc usr/bin
```

Comment:
I think you'll want the target to be an absolute path

Reply:
checked the base image (hive) ... the `workdir` is `/`

I can NOT use absolute path here

https://github.com/openshift/release/blob/8be771c27b5bbe59557f7cd92cdf3f7fe67bc7f9/ci-operator/config/openshift/ci-tools/openshift-ci-tools-master.yaml#L97

See https://github.com/openshift/ci-tools/pull/40

If everything goes through, I will give a try with absolute path here too.